From 06532c4eec22dae39932b405b43a63a9e0dc86e4 Mon Sep 17 00:00:00 2001
From: Enrico Jorns <ejo@pengutronix.de>
Date: Mon, 22 Aug 2016 16:03:02 +0200
Subject: [PATCH] script: introduce SCRIPT_CC

This alows to build target tools with different compiler call then
barebox itself.

Signed-off-by: Enrico Jorns <ejo@pengutronix.de>
---
 scripts/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/Makefile b/scripts/Makefile
index a3f6222..aa76046 100644
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -43,8 +43,8 @@ targetprogs-$(CONFIG_IMD_TARGET) += bareboximd-target
 # Let clean descend into subdirs
 subdir-	+= basic kconfig setupmbr
 
-quiet_cmd_csingle	= CC      $@
-      cmd_csingle	= $(CC) -Wp,-MD,$(depfile) $(CFLAGS) -o $@ $<
+quiet_cmd_csingle	= SCRIPT_CC      $@
+      cmd_csingle	= $(SCRIPT_CC) -Wp,-MD,$(depfile) $(CFLAGS) -o $@ $<
 
 __targetprogs	:= $(sort $(targetprogs-y) $(targetprogs-m))
 target-csingle	:= $(foreach m,$(__targetprogs),$(if $($(m)-objs),,$(m)))
-- 
2.8.1

